test/fixtures/language_models.yml

```diff
@@ -2,144 +2,176 @@
 
 gpt_4_1106_vision_preview:
   position: 11
-  name: gpt-4-1106-vision-preview
+  api_name: gpt-4-1106-vision-preview
   description: GPT-4 Turbo with Vision Preview (2023-11-06)
   supports_images: true
 
 gpt_4:
   position: 12
-  name: gpt-4
+  api_name: gpt-4
   description: GPT-4 (latest)
   supports_images: false
 
 gpt_4_0613:
   position: 13
-  name: gpt-4-0613
+  api_name: gpt-4-0613
   description: GPT-4 Snapshot improved function calling (2023-06-13)
   supports_images: false
 
 gpt_3_5_turbo:
   position: 14
-  name: gpt-3.5-turbo
+  api_name: gpt-3.5-turbo
   description: GPT-3.5 Turbo (latest)
   supports_images: false
 
 gpt_3_5_turbo_16k_0613:
   position: 15
-  name: gpt-3.5-turbo-16k-0613
+  api_name: gpt-3.5-turbo-16k-0613
   description: GPT-3.5 Turbo (2022-06-13)
   supports_images: false
 
 gpt_3_5_turbo_0125:
   position: 16
-  name: gpt-3.5-turbo-0125
+  api_name: gpt-3.5-turbo-0125
   description: GPT-3.5 Turbo (2022-01-25)
   supports_images: false
 
 gpt_3_5_turbo_1106:
   position: 17
-  name: gpt-3.5-turbo-1106
+  api_name: gpt-3.5-turbo-1106
   description: GPT-3.5 Turbo (2022-11-06)
   supports_images: false
 
 gpt_3_5_turbo_instruct:
   position: 18
-  name: gpt-3.5-turbo-instruct
+  api_name: gpt-3.5-turbo-instruct
   description: GPT-3.5 Turbo Instruct
   supports_images: false
 
 claude_3_opus_20240229:
   position: 19
-  name: claude-3-opus-20240229
+  api_name: claude-3-opus-20240229
   description: Claude 3 Opus (2024-02-29)
   supports_images: true
 
 claude_3_sonnet_20240229:
   position: 20
-  name: claude-3-sonnet-20240229
+  api_name: claude-3-sonnet-20240229
   description: Claude 3 Sonnet (2024-02-29)
   supports_images: true
 
 claude_3_haiku_20240307:
   position: 21
-  name: claude-3-haiku-20240307
+  api_name: claude-3-haiku-20240307
   description: Claude 3 Haiku (2024-03-07)
   supports_images: true
 
 claude_2_1:
   position: 22
-  name: claude-2.1
+  api_name: claude-2.1
   description: Claude 2.1
   supports_images: false
 
 claude_2_0:
   position: 23
-  name: claude-2.0
+  api_name: claude-2.0
   description: Claude 2.0
   supports_images: false
 
 claude_instant_1_2:
   position: 24
-  name: claude-instant-1.2
+  api_name: claude-instant-1.2
   description: Claude Instant 1.2
   supports_images: false
+
+camel:
+  user: keith
+  position: 25
+  api_name: camel
+  api_service: keith_service
+  description: gpt-3.5-turbo from fixtures
+  supports_images: false
+
+guanaco:
+  user: keith
+  position: 26
+  api_name: guanaco:large
+  api_service: keith_service
+  description: Guanaco (large)  from fixtures
+  supports_images: true
+
+alpaca:
+  user: taylor
+  position: 27
+  api_name: alpaca:medium
+  api_service: taylor_anthropic_service
+  description: Alpaca (large) from fixtures
+  supports_images: true
+
+pacos:
+  user: taylor
+  position: 28
+  api_name: pacos-imagine
+  api_service: taylor_openai_service
+  description: Lama pacos (large) from fixtures
+  supports_images: true
```

Comment:
Related to the comment I leave on api_service, this language_model fixture is seeded before each test, but we also use this fixture to seed new installations (seeds.rb) so everything in here will appear in production. This is because it’s really easy to keep inadvertently making code changes that break the app for new users; I’ve done it a bunch of times. My thinking has been: if I keep dev and test closer to production, then it’s easier to keep the app stable.

So basically, this new test data you added here will appear for new installs. Can you rewrite the tests to use the existing language model fixture data?

Reply:
See above comment, it looks like a stretch to combine test-user configuration and actual prod usage values.